src/App.css

```diff
@@ -0,0 +1,44 @@
+html,
+body,
+#root,
+.App,
+.grid-container {
+  height: 100%;
+  margin: 0;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 30px 1fr 1fr 30px;
+  grid-template-rows: 50px 30vw 1fr 30px;
+  gap: 1px 1px;
+  grid-template-areas:
+    "Menu-Bar Menu-Bar Menu-Bar       Menu-Bar"
+    ".        Blockly  Simulator      ."
+    ".        Blockly  Control-Panel  ."
+    "Footer   Footer   Footer         Footer";
+}
+
+.blockly-ui {
```

Comment:
Can you move this block to "Blockly.css" file and import it in the Blockly file?